app/locales/mr/locationTracking.json

```diff
@@ -0,0 +1,12 @@
+{
+    "start_logging": "स्टार्ट लॉगिंग",
+    "stop_logging": "स्टॉप लॉगिंग",
+    "logging_message": "हे सध्या दर पाच मिनिटांनी आपले लोकेशन खाजगीरित्या लॉग करत आहे. आपली लोकेशन माहिती आपला फोन सोडणार नाही.",
```

Comment:
no need for this: खाजगीरित्या . it does not translate the same way in your mind as in english...privately and securely
instead of सोडत नाही ... वापरला जाणार नाही (apli location maahiti aaplya sammati shivay vaparli jaanar nahi)